src/main/java/org/mozilla/jss/ssl/javax/JSSEngine.java

```diff
@@ -531,8 +544,9 @@ public void setEnabledCipherSuites(SSLCipher[] suites) throws IllegalArgumentExc
         ArrayList<SSLCipher> enabledCiphers = new ArrayList<SSLCipher>();
 
         for (SSLCipher cipher : SSLCipher.values()) {
+            if (!cipher.isSupported()) continue;
             try {
-                if (SSL.CipherPrefGetDefault(cipher.getID()) && cipher.isSupported()) {
+                if (SSL.CipherPrefGetDefault(cipher.getID())) {
```

Comment:
```suggestion
                if (cipher.isSupported() && SSL.CipherPrefGetDefault(cipher.getID())) {
```
This might be a nicer read? The latter won't be checked if `cipher.isSupported()` is false

Reply:
Sounds good. I updated the PR.